game/src/edit/select.rs

```diff
@@ -55,7 +57,10 @@ impl RoadSelector {
                     RewriteColor::ChangeAll(Color::hex("#4CA7E9")),
                 )
             } else {
-                Btn::svg_def("system/assets/tools/pencil.svg").build(ctx, "paint", Key::P)
```

Comment:
I think lots of this logic can get collapsed and use `.disabled()`. Right now the icons jump size when you select different tools:
![Screenshot from 2021-01-21 11-11-47](https://user-images.githubusercontent.com/1664407/105400194-7726e280-5bd9-11eb-9a41-a1bc2c59c640.png)
You can get to this panel by editing a lane, choosing edit multiple

Reply:
<img width="1185" alt="Screen Shot 2021-01-21 at 1 48 23 PM" src="https://user-images.githubusercontent.com/217057/105404239-5b2e3b80-5bef-11eb-8d78-0a38cba3918c.png">

Good catch - thank you! fixed.